_docs_operate/setup-connector-for-productive-use.md

```diff
@@ -8,16 +8,15 @@ toc: true
 properties:
   - id: SC063
   - category: Operations
-  - description: #1 https//enmeshed.eu/integrate/connector-installation
+  - description: #1 https://enmeshed.eu/integrate/connector-installation
```

Comment:
I would rather not insert any links under 'description', this only happens in about 4 places in the documentation. I would either leave this blank or insert text. (Here and in the Excel)